specification/maps/data-plane/Render/readme.md

```diff
@@ -32,6 +32,12 @@ tag: 2.1-preview
 add-credentials: true
 credential-default-policy-type: BearerTokenCredentialPolicy
 credential-scopes: https://atlas.microsoft.com/.default
+track2: true
+verbose: true
+sdk-integration: true
```

Comment:
I am good with it, but personally I am not sure on which codegen will this work (Java has it but only work in DPG without model).